### PR TITLE
MD-2882: Modified hive change metadata scripts

### PR DESCRIPTION
**Pull request by prasadns14**
Fixed the hive change metadata scripts, and the baseline accordingly. All test cases pass with hive 2.1 now.

@Agirish please review

### CODE REVIEW

File: framework/resources/Datasources/hive_storage/change_metadata2.ddl [hunk 1]
```diff
@@ -34,45 +34,45 @@ ALTER TABLE remove_columns2 REPLACE COLUMNS (
...
   int_col float,
-  bigint_col int,
+  bigint_col bigint,
   date_col varchar(15),
-  time_col timestamp,
-  timestamp_col date,
-  interval_col boolean,
+  time_col string,
+  timestamp_col timestamp,
+  interval_col string,
```
Comment (Agirish) [inline framework/resources/Datasources/hive_storage/change_metadata2.ddl]: Should interval be string, or long?

**Comment by Agirish** [comment]
+1 for 1.11.0-mapr. Before me merge into master, let's have another look to see if we are changing intended test behavior.

File: framework/resources/Datasources/hive_storage/change_metadata2.ddl (hunk 1 above)
Comment (prasadns14) [reply framework/resources/Datasources/hive_storage/change_metadata2.ddl]: When the table was created the data type assigned to interval was string, retained the same. 
Table is created in the change_metadata1.ddl

Comment (Agirish) [reply framework/resources/Datasources/hive_storage/change_metadata2.ddl]: cool, thanks! @rchallapalli will be the best person to share more details on the intended behavior.

**Pull request merged**